src/routes/route.py

```diff
@@ -91,11 +101,61 @@ def create_route_objects(
     total_population = _get_total_population(population_data, cars_per_person)
     result = []
     departure_times = _departure_times(total_population, start, end)
-    for route_path in tqdm(list_of_paths):
+    for origin_point in tqdm(origin_to_paths.keys()):
+        paths = origin_to_paths[origin_point]
         num_people_on_route = _get_num_people_on_route(
-            route_path, population_data, cars_per_person
+            origin_point, population_data, cars_per_person
         )
-        selected, departure_times = np.split(departure_times, [num_people_on_route])
-        route_object = Route(route_path, num_people_on_route, list(selected))
-        result.append(route_object)
+        number_of_diverse_routes = len(paths)
+        if number_of_diverse_routes < 1:
+            logging.warning(
+                f"Origin point {origin_point} has no routes to safety. Skipping."
+            )
+            continue
+        elif number_of_diverse_routes == 1 or num_people_on_route < len(paths):
```

Comment:
Why are these two cases treated as one? If num_people_on_route is 2 and len(paths) is 3 then i think we should put those 2 persons on the first two routes instead of both on the first route